### PR TITLE
Add vcsh conda recipe.

### PR DESCRIPTION
**Pull request by lebedov**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by lebedov** [comment]
This package's tests require several Perl packages that have not yet been packaged for conda-forge. Will I need to package all of them in order to get this package's tests to run, or can I just disable the package's tests?

**Comment by chrisburr** [comment]
There needs to be some kind of testing but it don't need to be the full test suite of the package. It can be as simple as `vcsh --help` or `test -f $PREFIX/bin/vcsh`. The aim is to verify the package isn't totally broken, any more advanced testing is just a nicety.

**Comment by lebedov** [comment]
Any thoughts as to how to address the build error on Windows? It seems that setting an env variable in the build script doesn't work in the same as on Linux.

File: recipes/vcsh/LICENSE [hunk 1]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (chrisburr) [inline recipes/vcsh/LICENSE]: I think this file can be removed as it is available from the source tarball.

Comment (lebedov) [reply recipes/vcsh/LICENSE]: OK.

**Comment by lebedov** [comment]
Not sure what to make of the latest error - I thought that one can run a program within the current directory by default on Windows.

**Comment by chrisburr** [comment]
It looks like `vcsh` is a shell script, does it support Windows?

**Comment by lebedov** [comment]
Yes, but I think cygwin would be necessary to provide bash. If that is an insufficient assumption for noarch conda-forge packages, perhaps it shouldn't be built for Windows.

File: recipes/vcsh/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{% set build = 0 %}
+{% set version = "1.20151229" %}
+
+package:
+    name: vcsh
+    version: {{version}}
+
+source:
+    fn: vcsh-{{version}}.tar.gz
```
Comment (chrisburr) [inline recipes/vcsh/meta.yaml]: This line isn't needed
```suggestion
```

**Comment by chrisburr** [comment]
If it is noarch package it will only be built on Linux so it's not a problem, it just won't be usable from Windows.

File: recipes/vcsh/meta.yaml (hunk 2 above)
Comment (lebedov) [reply recipes/vcsh/meta.yaml]: Updated.

**Comment by lebedov** [comment]
Should I add a build skip for Windows to placate AppVeyor?

**Comment by chrisburr** [comment]
No need and that would make the linter unhappy 😆

**Comment by lebedov** [comment]
Thanks!

**Pull request merged**